ui/app/AppLayouts/Communities/popups/TokenMasterActionPopup.qml

```diff
@@ -0,0 +1,178 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Core.Theme 0.1
+
+import AppLayouts.Communities.panels 1.0
+
+import utils 1.0
+
+
+StatusDialog {
+    id: root
+
+    enum ActionType {
+        RemotelyDestruct, Kick, Ban
+    }
+
+    title: {
+        if (actionType === TokenMasterActionPopup.ActionType.Ban)
+            return qsTr("Ban %1").arg(userName)
+        if (actionType === TokenMasterActionPopup.ActionType.Kick)
+            return qsTr("Kick %1").arg(userName)
+
+        qsTr("Remotely destruct TokenMaster token")
```

Comment:
`return` here? or does this work too? :thinking:

Reply:
It works (and bc of that it's easy to miss that) but `return` should be there imo anyways. Fixed.